API/Startup.cs

```diff
@@ -54,6 +63,11 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 endpoints.MapControllers();
             });
+
+            BackgroundJob.Enqueue(() => serverManager.RunPythonScripts());
+            BackgroundJob.Enqueue(() => serverManager.DeleteTranscodedFiles()); 
+            RecurringJob.AddOrUpdate(() => serverManager.DeleteTranscodedFiles(), Cron.Hourly);
+            RecurringJob.AddOrUpdate(() => serverManager.RunPythonScripts(), Cron.Daily);
```

Comment:
Das Enqueue ist doch redundant, weil wir sowieso einen RecurringJob einfügen, oder nicht?

Reply:
Stattdessen mit RecurringJob.Trigger direkt aufrufen
```suggestion
            RecurringJob.AddOrUpdate("DeleteOldFiles", () => serverManager.DeleteTranscodedFiles(), Cron.Hourly);
            RecurringJob.AddOrUpdate("UpdateDatabase", () => serverManager.RunPythonScripts(), Cron.Daily);
            RecurringJob.Trigger("DeleteOldFiles");
            RecurringJob.Trigger("UpdateDatabase");
```